setup.py

```diff
@@ -45,7 +45,7 @@
 
 setup(
     name='liota',
-    version='0.1.3',
+    version='0.1.4',
```

Comment:
The version in master branch is 0.1.2. I changed it to 0.1.3 for the refactoring branch. Let's keep it to 0.1.3 for the whole refactoring code.